saleor/payment/gateways/stripe/tests/conftest.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+
+from .....checkout import calculations
+from .....checkout.fetch import fetch_checkout_info, fetch_checkout_lines
+from .....plugins.manager import get_plugins_manager
+from .....plugins.models import PluginConfiguration
+from .... import TransactionKind
+from ....models import Transaction
+from ....utils import create_payment
+from ..plugin import StripeGatewayPlugin
+
+
+@pytest.fixture(scope="module")
+def vcr_config():
+    return {
+        "filter_headers": [("Authorization", "test_key")],
+    }
```

Comment:
Maybe we should remove it when we don't have tests with VCR.

Reply:
Right. I will remove it in the #7397. It will be easier for me ;)